src/views/Leads/Table/index.js

```diff
@@ -109,10 +109,18 @@ export default class Table extends React.Component {
             {
                 key: 'source',
                 order: 4,
+                modifier: ({
+                    sourceDetail = {},
+                    sourceRaw,
+                }) => sourceDetail.title || sourceRaw,
```

Comment:
We should check if sourceDetail is defined, instead of sourceDetail.title

Reply:
+1